pypit/arqa.py

```diff
@@ -715,10 +736,9 @@ def slit_profile(slf, mstrace, model, lordloc, rordloc, msordloc, textplt="Slit"
     return
 
 
-def slit_trace_qa(slf, frame, ltrace, rtrace, extslit, desc="", root='trace', outfil=None, normalize=True,
-                  use_slitid=None):
-    """
-    Generate a QA plot for the traces
+def slit_trace_qa(slf, frame, ltrace, rtrace, extslit, desc="",
+                  root='trace', outfil=None, normalize=True, use_slitid=None):
```

Comment:
Seems like we don't need to pass in `outfil` anymore. If we still need it, maybe it's best to rename this to `outfile` for consistency?